HW1.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+// Task #1
+console.log("1. Задача на повернення ініціалів для кожного імені з масиву, посортованих в алфавітному порядку:");
+
+const userNames = ["Петрик Ольга Іванівна", "Гнатюк Петро Антонович", 
+                    "Рудко Андрій Опанасович"];
+let initials;
+
+initials = userNames.map((str) => String(str).split(' '));
+
+for (let i=0; i<initials.length; i++){
+
+    initials[i] = initials[i].map((str) => String(str).slice(0,1)).join('.')+'.';
+}
+
+initials.sort();
+console.log(initials);
+
+// Task #2
+console.log("2. Задача на фільтрування масиву:");
+
+const userNames1 = ['Петро', 'Емма', 'Юстин', 'Ілля', 'Марта', 'Яна', 'Василь', 'Антон', 'Олена'];
+
+let filteredNames = [];
+const vowels = ['А', 'Е', 'Є', 'И', 'І', 'Ї', 'О', 'У', 'Ю', 'Я'];
+
+// 1st method
+console.log("через умовну конструкцію");
+
+for (let i=0; i<userNames1.length; i++){
+    if (vowels.includes(userNames1[i][0]))
+        filteredNames.push(userNames1[i]);
+}
+
+console.log(filteredNames);
+
+// 2nd method
+console.log("через вбудований метод масивів");
+
+filteredNames = userNames1.filter((el) => {return vowels.includes(el[0])});
```

Comment:
```suggestion
filteredNames = userNames1.filter((el) => vowels.includes(el[0]));
```
Якщо в один рядок, можна спростити синтаксис